docs/tutorials/kubernetes-basics/scale-intro.html

```diff
@@ -26,7 +26,7 @@ <h3>Objectives</h3>
             <div class="col-md-8">
        <h3>Scaling an application</h3>
 
-            <p>In the previous modules we created a <a href="https://kubernetes.io/docs/concepts/workloads/controllers/deployment/"> Deployment</a>, and then exposed it publicly via a <a href="https://kubernetes.io/docs/concepts/services-networking/service/">Service</a>. The Deployment created only one Pod for running our application.  When traffic increases, we will need to scale the application to keep up with  user demand.</p>
+            <p>In the previous modules we created a <a href="https://kubernetes.io/docs/concepts/workloads/controllers/deployment/"> Deployment</a>, and then exposed it publicly via a <a href="https://kubernetes.io/docs/concepts/services-networking/service/">Service</a>. The Deployment created only one Pod for running our application.  When traffic increases, we will need to scale the application to keep up with  user demands.</p>
```

Comment:
hi @huaxuan123 - I appreciate the thought, but the original english is more correct. Demand is treated as "singular" in this sentence, that's correct usage.